learning/inference.py

```diff
@@ -55,6 +56,44 @@ def parse_args():
     return arg_parser.parse_args()
 
 
+targets = [
+    # 710 between start and 1st target
+    [4940, 870],
+    # 940 between 1st and 2nd
+    [4000, 870],
+    # 470
+    [4000, 400],
+    # 3745
+    [255, 400],
+    # 2250
+    [255, -1850],
+    # 1080
+    [-825, -1850],
+    # 4335
+    [-825, 2485],
+    # 975
+    [150, 2485],
+    # 14,515 total distance for perfect path
```

Comment:
Better to use tuples since these values do not change or grow.

```suggestion
(4940, 870),
(4000, 870),
(4000, 400),
(255, 400),
(255, -1850),
(-825, -1850),
(-825, 2485),
(150, 2485),
```